checkstyle-tester/diff.groovy

```diff
@@ -321,9 +321,9 @@ def getProjectsStatistic(diffDir) {
                 def indexHtmlFile = new File(fileObjf.absolutePath + '/index.html')
                 indexHtmlFile.eachLine {
                     line ->
-                        if (line.matches(".*totalDiff\">[0-9]+.*")) {
-                            def totalDiff = Integer.valueOf(line.substring(line.indexOf('>') + 1,
-                                    line.lastIndexOf('<')))
+                        def lineMatcher = line =~ /totalDiff">(?<totalDiff>[0-9]++)/
```

Comment:
I do not understand this syntax, can you make it simple (java style) ?
I do not want to force ocntributrs to have extra knowledge to read this code. 

It is written in groovy, only because there is no scripts in java.